packages/binding-coap/test/coap-client-test.ts

```diff
@@ -30,15 +28,15 @@ should();
 @suite("CoAP client implementation")
 class CoapClientTest {
     @test async "should apply form information"() {
-        const testThing = Helpers.extend(
-            {
-                name: "Test",
-                properties: {
-                    test: {},
-                },
-            },
-            new ExposedThing(null)
-        );
+        // const testThing = Helpers.extend(
+        //     {
+        //         name: "Test",
+        //         properties: {
+        //             test: {},
+        //         },
+        //     },
+        //     new ExposedThing(null)
+        // );
```

Comment:
Reminder to ourselves, we should avoid commented code wherever is possible. (I don't blame you here cause I think is correct to comment it in the scope of this PR. Perhaps, can you open an issue pointing to this test? in that way we can review and implement a new test or decide to remove it once for all

Reply:
Done! - https://github.com/eclipse/thingweb.node-wot/issues/523
Yeah, here I've commented it out because the variable is created but never used (because the lines where it's used were already commented).